BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -179,26 +180,80 @@ public final class BlueprintView: UIView {
             )
         }
 
-        return element.content.measure(
-            in: measurementConstraint(with: size),
+        return sizeThatFits(measurementConstraint(with: fittingSize))
+    }
+
+    /// Measures the size needed to display the view within the given `SizeConstraint`.
+    /// by measuring the current `element` of the `BlueprintView`.
+    public func sizeThatFits(_ constraint: SizeConstraint) -> CGSize {
```

Comment:
internal?

Reply:
It felt reasonable to expose publicly; since the `SizeConstraint` through its types (eg, `Axis.unconstrained`) is clearer about how to use the provided `SizeConstraint` vs the regular `CGSize` magic numbers for unconstrained measurements.

Reply:
Yeah, I like this for the same reason as the original suggestion to make it explicit. That is, for consumers who know they have a blueprint view, they can do unconstrained measurement without jumping through greatest finite magnitude weirdness.